zhTW

OwenFeline opened the pull request:
Chinese（Traditional）

OwenFeline committed:
zhTW

Chinese（Traditional）

OwenFeline committed:
Update zhTW.lua

OwenFeline committed:
Update SoulshapeJournal.toc

OwenFeline committed:
Update zhTW.lua

christopheml commented:
Hello, thank you so much for providing a translation. 
I'll assign it to the 0.9 milestone. 
0.8 will see updates in translation strings, I'll notify you in the PR for the appropriate updates.

OwenFeline committed:
Merge branch 'christopheml:master' into master

OwenFeline committed:
Update zhTW.lua

for 0.8 version

Added：
　Goat Soul Guide
　Covenant Callings reward chests

Modified：
　Bunny Soul Guide
　Shoveltusk Soul Guide

christopheml commented:
Hello, there are a few translation key changes upcoming today for the 0.9 version, I'll start integrating translations after you have a chance to update the PR.

OwenFeline committed:
Merge branch 'christopheml:master' into master

OwenFeline committed:
Update for 0.9

Added：
　Normal (dungeon difficulty)
　Mythic (dungeon difficulty)

Removed：
　Normal or Heroic (dungeon difficulty)

OwenFeline committed:
Update zhTW.lua
